src/common/constants.ts

```diff
@@ -5,4 +5,6 @@ export const constants = {
     projectFileExtension: ".vott",
     assetMetadataFileExtension: "-asset.json",
     exportFileExtension: "-export.json",
+    projectSchemaVersion: "1.0.0",
```

Comment:
Thinking this version number should come from a separate package (vott-react?).  Otherwise it will be tricky to keep these in sync or to know when to update this schema version.